### PR TITLE
Fix plex-media-server Livecheck

### PR DESCRIPTION
**Pull request by bevanjkay**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@reitermarkus The livecheck for plex-media-server was failing, I have made the Regex a bit more flexible. I'm not sure if it meets the standard logic? Thanks.

File: Casks/plex-media-server.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   livecheck do
     url "https://plex.tv/api/downloads/5.json"
     strategy :page_match
-    regex(%r{href=.*?/PlexMediaServer-(\d+(?:\.\d+)*-\d+c\d+adfcb\d+)-x86_64\.zip}i)
+    regex(%r{href=.*?/PlexMediaServer-(\d+\.\d+\.\d+\.\d+-[0-9abcdef]+)-x86_64\.zip}i)
```
Comment (MindTooth) [inline Casks/plex-media-server.rb:13]: [a-z] not applicable?

Comment (bevanjkay) [reply Casks/plex-media-server.rb:13]: @MindTooth It's a file hash of sorts so it only seems to use a-f

Comment (vitorgalvao) [reply Casks/plex-media-server.rb:13]: `0-9` should never be used; that’s what `\d` is for. Also, `a-f` is better than `abcdef`.

**Review by vitorgalvao — DISMISSED** [review]

**Comment by bevanjkay** [comment]
@vitorgalvao sorry, I made a bit of a mess with this PR, when the version fell behind. But the checks are now passing.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**